src/tests/GfxBatchHbaoTest.cpp

```diff
@@ -0,0 +1,413 @@
+    {"cache-aware, defaults", "hbao-cache", false,
+     esp::gfx_batch::HbaoConfiguration{}.flags(),
+     esp::gfx_batch::HbaoConfiguration{}.intensity(),
+     esp::gfx_batch::HbaoConfiguration{}.radius(),
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness(), 0.0f, 0.0f},
+    {"cache-aware, AO special blur", "hbao-cache-sblur", false,
+     esp::gfx_batch::HbaoConfiguration{}.flags() |
+         esp::gfx_batch::HbaoFlag::UseAoSpecialBlur,
+     esp::gfx_batch::HbaoConfiguration{}.intensity(),
+     esp::gfx_batch::HbaoConfiguration{}.radius(),
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness(), 1.0f, 0.1f},
+    {"cache-aware, strong effect", "hbao-cache-strong", false,
+     esp::gfx_batch::HbaoConfiguration{}.flags(),
+     esp::gfx_batch::HbaoConfiguration{}.intensity() * 2.0f,
+     esp::gfx_batch::HbaoConfiguration{}.radius() * 1.5f,
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness() * 5.0f, 0.0f, 0.0f},
+    {"cache-aware, strong effect, AO special blur", "hbao-cache-strong-sblur",
+     false, esp::gfx_batch::HbaoConfiguration{}.flags(),
+     esp::gfx_batch::HbaoConfiguration{}.intensity() * 2.0f,
+     esp::gfx_batch::HbaoConfiguration{}.radius() * 1.5f,
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness() * 5.0f, 1.0f, 0.1f},
+    {"cache-aware, layered with image load store", "hbao-cache-layered", false,
+     esp::gfx_batch::HbaoConfiguration{}.flags() |
+         esp::gfx_batch::HbaoFlag::LayeredImageLoadStore,
+     esp::gfx_batch::HbaoConfiguration{}.intensity(),
+     esp::gfx_batch::HbaoConfiguration{}.radius(),
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness(), 0.0f, 0.0f},
+    {"cache-aware, layered with image load store, AO special blur",
+     "hbao-cache-layered-sblur", false,
+     esp::gfx_batch::HbaoConfiguration{}.flags() |
+         (esp::gfx_batch::HbaoFlag::LayeredImageLoadStore |
+          esp::gfx_batch::HbaoFlag::UseAoSpecialBlur),
+     esp::gfx_batch::HbaoConfiguration{}.intensity(),
+     esp::gfx_batch::HbaoConfiguration{}.radius(),
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness(), 1.0f, 0.1f},
+    {"cache-aware, layered with geometry shader", "hbao-cache-geom", false,
+     esp::gfx_batch::HbaoConfiguration{}.flags() |
+         esp::gfx_batch::HbaoFlag::LayeredGeometryShader,
+     esp::gfx_batch::HbaoConfiguration{}.intensity(),
+     esp::gfx_batch::HbaoConfiguration{}.radius(),
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness(), 0.0f, 0.0f},
+    {"cache-aware, layered with geometry shader, AO special blur",
+     "hbao-cache-geom-sblur", false,
+     esp::gfx_batch::HbaoConfiguration{}.flags() |
+         (esp::gfx_batch::HbaoFlag::LayeredGeometryShader |
+          esp::gfx_batch::HbaoFlag::UseAoSpecialBlur),
+     esp::gfx_batch::HbaoConfiguration{}.intensity(),
+     esp::gfx_batch::HbaoConfiguration{}.radius(),
+     esp::gfx_batch::HbaoConfiguration{}.blurSharpness(), 1.0f, 0.1f},
```

Comment:
As we discussed previously, might make sense to add the GS + image load/store variants again but compare them to the default `"hbao-cache"` images with a higher threshold, to be sure they don't diverge too much.